Index: .github/workflows/androidDemoNightly.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>name: Nightly build at 11pm Android Pruvit Pulse Demo \r\n\r\non:\r\n  workflow_dispatched:\r\n\r\n#  schedule:\r\n#    - cron: '0 23 * * *'\r\n\r\n\r\njobs:\r\n  nightly_build:\r\n    runs-on: windows-latest\r\n    steps:\r\n      - name: Checkout\r\n        uses: actions/checkout@v3.3.0\r\n      - name: Install and Run Appium Server\r\n        uses: moatazeldebsy/appium-server-gitHub-action@V1.0.4\r\n         \r\n      - name: Run Unit Tests\r\n        run: mvn test\r\n      \r\n      - name: Build with Maven\r\n        run: mvn -DskipTests -B package --no-transfer-progress --file ./pom.xml\r\n\r\n      - uses: actions/upload-artifact@master\r\n        with:\r\n          name: digibank.war\r\n          path: ./target/*digitalbank*.war\r\n          \r\n            \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.github/workflows/androidDemoNightly.yml b/.github/workflows/androidDemoNightly.yml
--- a/.github/workflows/androidDemoNightly.yml	(revision 6429b56ac9da6a81a60460fe90f3ec85356ca0d7)
+++ b/.github/workflows/androidDemoNightly.yml	(date 1689863286428)
@@ -2,9 +2,8 @@
 
 on:
   workflow_dispatched:
-
-#  schedule:
-#    - cron: '0 23 * * *'
+  schedule:
+    - cron: '0 23 * * *'
 
 
 jobs:
